### ISSUE TITLE
popunk_sketch :: not found

### ISSUE THREAD

**Issue by EricDeveaud** [johnlees/PopPUNK#114]
trying to run the tests

```
Singularity> python3 run_test.py 
Running database creation + DBSCAN model fit + fit refinement (--easy-run)
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
/bin/sh: 1: poppunk_sketch: not found
Traceback (most recent call last):
  File "../poppunk-runner.py", line 10, in <module>
    main()
  File "/root/PopPUNK-2.2.0/PopPUNK/__main__.py", line 288, in main
    dbFuncs = setupDBFuncs(args, kmers, args.min_kmer_count, qc_dict)
  File "/root/PopPUNK-2.2.0/PopPUNK/utils.py", line 56, in setupDBFuncs
    version = checkSketchlibVersion()
  File "/root/PopPUNK-2.2.0/PopPUNK/sketchlib.py", line 51, in checkSketchlibVersion
    version = line.rstrip().decode().split(" ")[1]
IndexError: list index out of range
```

**Comment by johnlees** [comment]
I think this is because you haven't got pp-sketchlib installed, and changed the import in #113

**Comment by johnlees** [comment]
Resolved at #113
